en/configure-a-tidb-cluster.md

```diff
@@ -433,7 +433,7 @@ spec:
       logLevel: info
 ```
 
-For all configurable start parameters of TiCDC, see [TiCDC configuration](https://github.com/pingcap/ticdc/blob/master/cmd/ticdc.toml).
+For all configurable start parameters of TiCDC, see [TiCDC configuration](https://github.com/pingcap/tiflow/blob/master/cmd/ticdc.toml).
```

Comment:
404 reported for `https://github.com/pingcap/tiflow`, it's a private repo?

Reply:
Same 404 here. Please do not merge until the repo is publicly accessible.